docs/en/reference/batch-processing.rst

```diff
@@ -75,7 +75,7 @@ Iterating results
 ~~~~~~~~~~~~~~~~~
 
 An alternative solution for bulk updates is to use the
-``Query#getIterable()`` facility to iterate over the query results step
```

Comment:
The docs still need change as `getIterable` needs to be removed form the sentence.

Reply:
I fixed the naming here https://github.com/doctrine/orm/pull/8293 though there are no reviews yet. It is not part of this PR.